Vlv2TbltDevicePkg/PlatformPkgGcc.fdf

```diff
@@ -160,10 +160,11 @@ FV = FVMAIN_COMPACT
   #
 $(FLASH_REGION_RESERVED_OFFSET)|$(FLASH_REGION_RESERVED_SIZE)
 gPlatformModuleTokenSpaceGuid.PcdFlashRsvdBase|gPlatformModuleTokenSpaceGuid.PcdFlashRsvdSize
-DATA = {
-  0x52, 0x45, 0x53, 0x45, 0x52, 0x56, 0x45, 0x44,
-  0x00
-}  # "RESERVED" in HEX
+FILE = hello_world.efi
+#DATA = {
```

Comment:
Since this branch is intended to launch target binaries via trampoline, these DATA sections are not necessary